db/migrate/20220601235048_create_posts.rb

```diff
@@ -0,0 +1,14 @@
+class CreatePosts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.integer :user_id
+      t.string :title
+      t.string :text
```

Comment:
- [ ] The text should be with the type of `text` because it may be longer please update that

Reply:
Thanks